recipes/semimap/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "semimap" %}
+# NOTE: Semimap does not have a release or a version.
+#
+# We rely on the last commit on upstream at the time of writing
+# and simply use it as the version identifier.
+#
+# Q: Is there a better alternative?
+{% set version = "9a106db4a8eb38a5b6e7c94a2a7addf8de7f6dd2" %}
+{% set organization = "QuantStack" %}
+{% set sha256 = "4d24673f4b83d8de1e573ef7cd06060e3e1a765e272f15a8d7d9a470f2a80b0e" %}
```

Comment:
To me, the last remaining piece of complexity to address is to have a proper version identifier for semimap.

https://github.com/QuantStack/semimap/commit/3e2175dbaf3f17004f91e0a0f26de39665607241 somewhat has introduced a v.1.0.0 version; after the first review of this recipe, the repository was restructured to use cmake.

Should we bump the version number, use a tag and create a release before this recipe gets approved? What do you think, @JohanMabille?

Reply:
Yes, we should tag a release on the current commit (since the version is already set) and use it for this PR.

Reply:
What would be an appropriate version identifier? v1.1.0?